newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -194,6 +175,13 @@ export default function SummaryPage() {
                   <Button className={styles['create-event-button']}>
                     <Trans>Create event</Trans>
                   </Button>
+                  <Button
+                    color="red"
+                    className={styles['delete-button']}
+                    onClick={() => setDeletionModalOpen(true)}
+                  >
+                    <Trans>Delete newdle</Trans>
+                  </Button>
```

Comment:
I don't really like the placement of the delete button in case of a finalized newdle:

![image](https://user-images.githubusercontent.com/179599/97091853-4caba480-163f-11eb-8aae-10965d0bba38.png)

- It now requires wrapping the text in the buttons, which doesn't look great
- It looks like an action that's related to the slot, not the whole newdle

I would move it back to its original place below the list of slots (ie where it was before this PR).